.bookignore

```diff
@@ -2,7 +2,7 @@ build/
 config/
 dist/
 fonts/
-images/
+./images/
```

Comment:
No, I'm still on El Capitan. This is a weird one - my guess is that we're somehow not using the same version of Gitbook, even though I removed my global installation and did a `rm -rf` and reinstallation of local modules.